rnnalyse/activations/activations_reader.py

```diff
@@ -63,11 +64,13 @@ def create_data_split(self,
         split = int(self.data_len * train_test_split)
 
         n = split if train_subset_size == -1 else train_subset_size
-        train_indices = np.random.choice(range(n), n, replace=False)
+        indices = np.random.choice(range(self.data_len), self.data_len, replace=False)
```

Comment:
It would actually be good to write a unittest for this.

Reply:
How this behavior works could probably at least be specified in a docstring

Reply:
Yes there should be some documentation indeed. And probably change the name, if we don't even have the same idea about what it is supposed to do that's not good XD.
For me, having a functionality that lets you specify the exact amount of training points is not very useful, what I would find more useful is being able to not use the entire dataset that was is in the file. If you want to do a small test, for instance, you can then just specify that from the commandline, instead of having to create a new, smaller, file. @jumelet and @Kaleidophon, what do you think of changing the name of `train_subset_size` to `data_subset_size`, the interpretation is then a bit more intuitive (similar to 'cutoff' in the extract script).

Reply:
Mh, tbh I don't have an opinion of what would be more useful. @jumelet ?

Reply:
Ak then question: in the current formulation, aren't they just the same formulations of a different thing? If you define `train_subset size` this decides already the split, and if you define `train_test_split' this determines the subset size of the training data. 

Of course, sometimes it can be very useful to have two ways to specify the same thing because one specification may be more easily accessible, but in this case I don't really see that happening.
@Kaleidophon if you find these two different ways of specifying how much of the data to use for training useful, I'm curious what kind of situations do you think it would be more convenient to use `train_subset_size` instead of the split. Maybe I'm not considering them.

Reply:
I just see them as a relative and an absolute way to specify the split. I could see that someone might want to say "I want to use _exactly_ 1000 samples to train my model" but I also acknowledge that usually it's done by giving a percentage. That being said, I don't mind leaving it there just in case someone needs it but I am also not attached to it and open to remove or replace it.

Reply:
Then I propose to do it as I suggested before, and use the argument `data_subset_size` as a  way for people to only use part of their dataset, similar to the `cutoff` argument in the extractor

Reply:
Done (with tests)!